src/routes/Gift.tsx

```diff
@@ -67,7 +67,9 @@ function InboundWarning() {
         const network = state.mutiny_wallet?.get_network() as Network;
 
         const threshold = network === "bitcoin" ? 100000 : 10000;
-        const balance = state.balance?.lightning || 0n;
+        const balance =
+            (state.balance?.lightning || 0n) +
+            (state.balance?.federation || 0n);
```

Comment:
we are doing this a bunch everywhere, probably a good indication we should add in mutiny-core something like `can_pay_lightning(amt)` that checks both federation and lightning balance. Good for a follow up though